mmdet/models/detectors/rpn.py

```diff
@@ -1,6 +1,6 @@
 import mmcv
 import torch
-from mmcv.image import tensor2imgs
+from mmcv.image import tensor2imgs,imwrite
```

Comment:
imwtite seems to be unnecessary here.

Reply:
Thanks for your reminding. I deleted it.